.github/workflows/nodered-js.yml

```
@@ -0,0 +1,49 @@
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: ./.github/actions/setup-go
+        with:
+          go_version: ${{ env.GO_VERSION }}
+      - name: Install Ginkgo
+        run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.19.0
+      - name: Test
+        run: TEST_NODERED_JS=1 make test 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)